src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -58,33 +53,127 @@ If you'll need to do a good amount of partitioning of users' access to accounts
 * Customers with smaller or flatter organizations that require transparency for all teams across all of their data could have as little as one or two groups.
 * For organizations where all data is in a single account, a common configuration may be around five or six groups.
 
-## Use the user migration wizard [#find]
+## Migrate your users with the migration wizard [#find]
 
 If you meet [the requirements](#requirements), you can use the wizard. To start using the migration wizard: 
 
-1. From [one.newrelic.com](https://one.newrelic.com), click **Apps** in the top navigation and then click the **User migration walkthrough** app.
-2. Optional: if you have problems, see [Tips](#tips).
+1. From [one.newrelic.com](https://one.newrelic.com), click **Apps** in the top navigation. 
+2. In the table of apps, click the **User migration walkthrough** app.
+3. Optional: If you want more help and context, see the sections below for tips and recommendations for specific migration wizard pages. 
 
-Once your users are migrated to the new model, you can find and manage them by clicking the account dropdown, clicking **Administration**, and using these UI pages: 
+If you've successfully completed migration, learn [how to manage your users](#manage-users). 
 
-* **User management**: use this to view and add users, change their type (basic vs. full), change their group, and approve user upgrade requests. 
-* **Organization and access**: use this to create access grants (granting groups access to roles and accounts), and configure authentication domains (SAML SSO settings and SCIM settings, and more). 
+## Step 1: Create admins [#page1]
 
-For more about these tools, see the [new user management docs](/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-managing-users/).
+![User migration page 1](./images/user-migration-page-1.png "user-migration-page-1.png")
```

Comment:
This image is missing a caption.

Reply:
i was not using captions for most of them, as it felt pretty obvious I was just showing images from the walkthrough.

Reply:
Also, I think this image might be easier to see if you crop out the Migration FAQ from the right side of the image.